include/deal.II/lac/generic_linear_algebra.h

```diff
@@ -68,10 +68,51 @@ namespace LinearAlgebraPETSc
 {
   using namespace dealii;
 
+  /**
+   * Typedef for the base class for preconditioners.
+   */
+  typedef PETScWrappers::PreconditionerBase PreconditionerBase;
+
+  /**
+   * Typedef for the base class for linear solvers.
+   */
+  typedef PETScWrappers::SolverBase SolverBase;
```

Comment:
what would you actually do with these class names? You can't create an object of `SolverBase` or `PreconditionerBase`...